packages/portfolio-contract/src/portfolio.flows.ts

```diff
@@ -681,6 +677,22 @@ const stepFlow = async (
     }
   }
 
+  const acctDests = [
+    ...new Set(
+      moves.map(({ dest }) => getChainNameOfPlaceRef(dest)).filter(Boolean),
```

Comment:
Is this listing all touched accounts or only the ones we actually need to go create?  Looks like the former.